upstairs/src/guest.rs

```diff
@@ -34,11 +34,6 @@ use uuid::Uuid;
  */
```

Comment:
Is this comment out of date now that `ds_id` is gone here?

Reply:
It's kinda weird; edited to read `JobId` instead of `ds_id` (because we still have a `HashMap<JobId, GtoS>`